app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesViewHolder.kt

```diff
@@ -78,20 +75,21 @@ class PocketStoriesViewHolder(
             }
         }
 
-        Column(modifier = Modifier.padding(top = 72.dp)) {
-            SectionHeader(
-                text = stringResource(R.string.pocket_stories_header_1),
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(horizontal = horizontalPadding)
-                    .wrapContentHeight(align = Alignment.Top)
+        Column(
+            modifier = Modifier.padding(
+                top = 72.dp,
+                start = horizontalPadding,
+                end = horizontalPadding
```

Comment:
Can you add the horizontal padding via https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/home/collections/CollectionViewHolder.kt#45? When we convert the recyclerview into a column, we would expect the parent column of this component to set the padding. So, I would rather we not set the horizontal padding in the component in order to future proof ourselves.